### ISSUE TITLE
commentedOutCode: false negative if preceded by prose

### ISSUE THREAD

**Issue by rittneje** [go-critic/go-critic#1170]
The `commentedOutCode` check will properly flag this:
```go
// fmt.Println()
```

But it will not flag this:
```go
// print a newline
// fmt.Println()
```

**Comment by cristaloleg** [comment]
Because 1st comment is just a code and 2nd is text+code. 

I know that might be suspicious but just a commented code might be a leftover that should be removed, but a commented code with a text (an explanation like in your example) can be a good thing to keep.
